tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,180 @@
+        settings.open_sound()
+        self.take_screenshot('sound')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_display()
+        self.take_screenshot('display')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_homescreen()
+        self.take_screenshot('homescreen')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_search()
+        self.take_screenshot('search')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_navigation()
+        self.take_screenshot('navigation')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_notification()
+        self.take_screenshot('notification')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_date_and_time()
+        self.take_screenshot('date_and_time')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_language()
+        self.take_screenshot('language')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_keyboard()
+        self.take_screenshot('keyboard')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_themes()
+        self.take_screenshot('themes')
+        settings.return_to_prev_menu(settings.screen, gaia_header=False)
+        
+        settings.open_addons()
+        self.take_screenshot('addons')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_achievements()
+        self.take_screenshot('achievements')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_firefox_accounts()
+        self.take_screenshot('firefox_accounts')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_findmydevice()
+        self.take_screenshot('findmydevice')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_screen_lock()
+        self.take_screenshot('screen_lock')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_app_permissions()
+        self.take_screenshot('app_permissions')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_do_not_track()
+        self.take_screenshot('do_not_track')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_browsing_privacy()
+        self.take_screenshot('browsing_privacy')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_media_storage()
+        self.take_screenshot('media_storage')
+        settings.return_to_prev_menu(settings.screen)
+        
+        settings.open_application_storage()
+        self.take_screenshot('application_storage')
+        settings.return_to_prev_menu(settings.screen)
+
+        ### Device Information and its sub pages
```

Comment:
I thought once we populate this script with the sub-page actions, it would become a bit harder to see where each settings menu is tested.  I thought I could make it more discernable by putting a line with #'s after each of them, do you think it's necessary?